x/payments/internal/types/events.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+const (
+	EventTypePaymentContractAuthorisation = "payment_contract_authorisation"
+	EventTypeCreatePaymentTemplate        = "create_payment_template"
+	EventTypeCreatePaymentContract        = "create_payment_contract"
+	EventTypeCreateSubscription           = "create_subscription"
+	EventTypeGrantDiscount                = "grant-discount"
```

Comment:
```suggestion
	EventTypeGrantDiscount                = "grant_discount"
```

Reply:
code has been added from EventTypeGrantDiscount                = "grant-discount" to  EventTypeGrantDiscount                = "grant_discount"